server/models/Collective.js

```diff
@@ -1207,6 +1208,19 @@ Collective.prototype.deactivateAsHost = async function () {
     );
   }
 
+  // Make sure we clean up all pending applications
+  await models.HostApplication.update(
+    { status: HostApplicationStatus.EXPIRED },
+    { where: { HostCollectiveId: this.id } },
+  );
+
+  await models.Member.update({ deletedAt: new Date() }, { where: { MemberCollectiveId: this.id, role: 'HOST' } });
```

Comment:
Do we really need that one? `HOST` member are not supposed to be created until the application gets approved

Reply:
@Betree : I've double checked this and it seems that we do add an entry to the `Member` table upon applying for the host; 

https://github.com/opencollective/opencollective-api/blob/b41f19b6509d12289772f39b9c1b1b3bf6f64058/server/models/Collective.js#L2343-L2348

Reply:
Interesting :+1: